src/components/GuessResult.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useCallback } from "react";
+import { RouteComponentProps } from "@reach/router";
+import { GameContext, GameEvent } from "gameStateMachine";
+import { Interpreter } from "xstate";
+import { useService } from "@xstate/react";
+import { Wrapper, NextButton } from "./GuessResult.styles";
+
+interface GuessResultProps extends RouteComponentProps {
+    gameService: Interpreter<GameContext, any, GameEvent, any>;
+}
+
+const GuessResult: React.FC<GuessResultProps> = ({ gameService }) => {
+    const [state, send] = useService(gameService);
+
+    const handleClick = useCallback(() => {
+        if (
+            state.matches({ playing: { answer: "incorrectLast" } }) ||
+            state.matches({ playing: { answer: "correctLast" } })
+        ) {
+            send({ type: "COMPLETE" });
+        }
+        send({ type: "NEXT_ROUND" });
+    }, [state, send]);
+
+    const correctAnswerMessage = "That's right! You really know your stuff...";
+    const incorrectAnswerMessage =
+        "Nope, that's not it - are you sure you're a fan?";
+
+    const incompleteButtonText = "Another, please!";
+    const completeButtonText = "Show me!";
+
+    let resultMessage: string = "";
+    let buttonText: string = "";
+    let song: any;
+    let correct: boolean = false;
+    let complete: boolean = false;
+
+    if (state.matches({ playing: { answer: "correct" } })) {
+        resultMessage = correctAnswerMessage;
```

Comment:
These things might be something you can store with the state machine. xstate has the concept of [state meta data](https://xstate.js.org/docs/guides/states.html#state-meta-data). Which lets you associate some static data with a given state. 

The same should apply to things like "correct" and "complete" the state machine already knows these things, so here we've taken the output of the machine and converted them back into local state that *could* result in an incorrect combination of items (e.g what happens if we accidentally set correct to false but resultMessage to incorrectAnswerMessage)